EMML/1001-2000/EMML1832.xml

```diff
@@ -45,35 +146,930 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
+                           ይምሐሮ፡ እግዚአብሔር፡ በዝኒ፡ መዋዕል፡ ወበዘ፡ ይመጽእ፡ ዓለም፡ ለገብሩ፡ ኢየሱስ፡ ሞአ። <hi rend="rubric">ቂርቆስ፡ ወእሙ፡ </hi></q>
+                     </decoNote>
+                     <decoNote xml:id="d13" type="miniature">
+                        <locus target="#16v"/>
+                        <desc>Miniature depicting <persName ref="PRS3337Daniel"/> in the lion's den being served bread by 
+                           <persName ref="PRS4914Habakkuk"/>.</desc>
+                        <q xml:lang="gez">ወነዋ፡ እንባቆም፡ ዘአውረደ፡ ኅብስተ፡ ለዳንኤል፡ ወመልኣከ፡ እግዚአብሔር፡ እኂዞ፡ በሥዕርተ፡ ርእሱ፡ እንዘ፡ እትትረኀው፡ ግብ። ። 
+                           ወወሀብዎሙ፡ መብልዐ፡ ለእለ፡ ይብልዑ፡ ለውእቱ፡ መብልዕ፡ ሰገዱ፡ እለ፡ ይበልዑ፡ ወነዋ፡ ዳንኤል፡ ውስተ፡ ግበ፡ ኣናብስት። ። </q>
+                     </decoNote>
+                     <decoNote xml:id="d14" type="miniature">
+                        <locus target="#17r"/>
+                        <desc>Miniature depicting <persName ref="PRS11288AnAzMi"/> in flames being protected by the angel of God.</desc>
+                        <q xml:lang="gez">ሲድራቅ፡ ወሚሳቅ፡ ወአብደናጎ፡ ወመልአከ፡ እግዚአብሔር፡ ይዓቅቦሙ፡ በማእከለ፡ እቶን፡ እሳት። ።</q>
+                     </decoNote>
+                     <decoNote xml:id="d15" type="miniature">
+                        <locus target="#17v"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1002Annunciation"/> and
+                           <ref type="authFile" corresp="AT1004Bitter"/>.</desc>
+                        <q xml:lang="gez">ኀበ፡ ዜነዋ፡ ገብርኤል፡ ለማርያም፡ ቅድስት፡           ኀበ፡ አስተያ፡ ዮሴፍ፡ ለማርያም፡ ማየ፡ ዘለፋ። </q>
+                     </decoNote>
+                     <decoNote xml:id="d16" type="miniature">
+                        <locus target="#18r"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1025NativityJesus"/> and the shepherds seeing the star.</desc>
+                        <q xml:lang="gez">ኀበ፡ ተወልደ፡ ክርስቶስ፡ ለህም፡ ወአድግ፡ ሰገዱ፡       ኖሎት፡ ኀበ፡ ርእዩ፡ ኮከብ፡ </q>
+                     </decoNote>
+                     <decoNote xml:id="d17" type="miniature">
+                        <locus target="#18v"/>
+                        <desc>Miniature depicting <persName ref="PRS6462MagiWi"/> offering their presents to Christ.</desc>
+                        <q xml:lang="gez">ኀበ፡ አብአ፡ አምኃ፡ ሰብኣ፡ ሰገል፡ 
+                           ወመልአከ፡ አግዚአብሔር፡ ቆመ፡ መልዕልቴሆሙ፡ ኀበ፡ ሀሎ፡ ሕፃን፡ ምስለ፡ እሙ፡ </q>
+                     </decoNote>
+                     <decoNote xml:id="d18" type="miniature">
+                        <locus target="#19r"/>
+                        <desc>Miniature depicting <ref type="authFile" corresp="AT1016Flight"/>.</desc>
+                        <q xml:lang="gez">ዮሴፍ፡ ነሥአ፡ ሕፃነ፡ ወእም፡ ወሐረ፡ ብሔረ፡ ግብጸ፡</q>
+                     </decoNote>
+                     <decoNote xml:id="d19" type="miniature">
+                        <locus target="#19v"/>
+                        <desc>Miniature depicting <persName ref="PRS00000000000">Rachel</persName> and another woman weeping for their 
```

Comment:
did not you want to create and ID for Rachel (old testament patriarch)? You can provisionally refer directly to wd:Q207389

Reply:
Do you mean to refer to wd:Q207389 in a new PRS ID or to refer to wd:Q207389 directly in the manuscript record?

Reply:
See example 2 https://betamasaheft.eu/Guidelines/?id=named-entities-and-authority-lists

Reply:
(but no mark up is also ok)

Reply:
Thanks! Done.